src/main/java/com/lambdaherding/edi/ch02/ThreadLocalExample.java

```diff
@@ -22,7 +42,12 @@ private static void show( long epochMillis ) {
 	}
 
 	public static void main( String[] args ) {
-		ThreadLocalExample.show( 31, Month.MARCH, 2017 );
-		ThreadLocalExample.show( 1491004800_000L );
+		for ( int i = 1; i <= 3; i++ ) {
+			final int day = i; // Must be final or effectively final!
+			new Thread( () -> {
+				ThreadLocalExample.show( day, Month.MARCH, 2017 );
+				ThreadLocalExample.show( 1491004800_000L * new Long(day) ); // little bit silly, but shows up nicely in the comparative output
```

Comment:
Swithering over whether we should change this to non-lambda style threads or not. I guess that's not really the point of the exercise, but it's perhaps a neat little bonus for future devs.